decidim-core/lib/decidim/core.rb

```diff
@@ -587,6 +587,37 @@ def self.reset_all_column_information
     {}
   end
 
+  def self.open_data_manifests
+    [
+      OpenStruct.new(
+        name: :moderated_users,
+        collection: lambda { |organization|
+                      Decidim::UserModeration.joins(:user).where(decidim_users: { decidim_organization_id: organization.id }).where.not(decidim_users: { blocked_at: nil })
+                    },
+        serializer: Decidim::Exporters::OpenDataBlockedUserSerializer,
+        include_in_open_data: true
+      ),
+      OpenStruct.new(
+        name: :moderations,
+        collection: ->(organization) { Decidim::Moderation.where(participatory_space: organization.participatory_spaces).includes(:reports).hidden },
```

Comment:
I agree that we should only show the `hidden` moderations. Can you add a spec for non-hidden moderations (that shouldn't be available in the CSV file) please?

Reply:
done in 7c7c245f67